spark/v4.0/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestRewriteManifestsProcedure.java

```diff
@@ -265,7 +266,7 @@ public void testRewriteSmallManifestsWithoutCaching() {
         .hasSize(1);
   }
 
-  @TestTemplate
+  @Disabled // Spark SQL does not support case insensitive for named arguments
```

Comment:
Strictly speaking, it's a regression, but I think it's okay to respect the Spark SQL syntax.

Reply:
oh, there is no spark config for this, right?

Reply:
How do built-in functions work in Spark? Do procedures behave consistently with that?

Reply:
If I remember correctly, Spark should share arg matching with functions.

Reply:
I think we should just follow what Spark does today.

Reply:
If Spark decides to change this behavior both for functions and procedures in the future, that's fine. I don't think we have to worry about this in Iceberg. Also, this will be a brand-new Iceberg jar for Spark 4.0.

Reply:
I vote disable case insensitivity for now and merge this pr in 1.10 to avoid breaking things in 1.11